geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/AbstractPartitionedRepositoryManager.java

```diff
@@ -111,6 +121,10 @@ protected IndexRepository computeRepository(Integer bucketId) {
     return repo;
   }
 
+  protected void allowRepositoryComputation() {
```

Comment:
ah ok, makes more sense, I think my mind mistook this method name with allowing the repo to do computation rather than allowing computing of repository